snmp/tests/test_e2e_core_profiles/utils.py

```diff
@@ -32,6 +32,8 @@ def assert_extend_generic_if(aggregator, common_tags):
 
 
 def assert_extend_generic_ip(aggregator, common_tags):
+    """Add the following to the snmprec
+    1.3.6.1.2.1.4.31.1.1.4.1|2|310637142"""
```

Comment:
```suggestion
    """Add the following to the snmprec
1.3.6.1.2.1.4.31.1.1.4.1|2|310637142
    """
```

What about not indenting, it makes easier to copy paste

Reply:
the linter doesn't accept it